lesson06/assignment/Drama.html

```diff
@@ -0,0 +1,114 @@
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
+        <link rel="stylesheet" href="css/style.css" />
+        <title>Assignment 06</title>
+    </head>
+    <body>
+        <header>
+          <div class="grid-container">
+                <div class="header-left">
+                    <div class="header-left-1">
+                      <h1>BOOK MY EVENT</h1>   
+                    </div>
+                    <div class="header-left-2">
+                      <form method="get">
+                        <input type="text" name="q" placeholder="search for events">
+                        <input type="submit" value="search">
+                      </form>
+                    </div>
+                </div>
+                <div class="header-right">
+                    <label>Location<br>
+                        <select name="location">
+                            <option value="Seattle">Seattle</option>
+                            <option value="Bellevue">Bellevue</option>
+                            <option value="Tacoma">Tacoma</option>
+                        </select>
+                    </label>
+                </div>
+            
+     
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
+            <div class="collapse navbar-collapse" id="navbarNav">
+                <ul class="navbar-nav">
+                    <li class="nav-item">
+                        <a class="nav-link" href="index.html">Home</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="Drama.html">Drama</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="comedyshows.html">Comedy shows</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="kids.html">Kids</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+  
+            <section>
+                    <h3>Filters</h3>
+                    <nav>
+                        <ul>
+                          <li>Language
+                            <ul class="menu-1">
+                              <li class="menu-1__li">English</li>
+                              <li>Spanish</li>
+                              <li>French</li>
+                              <li>Japanese</li>
+                              <li>Korean</li>
+                            </ul>
+                          </li>
+                        </ul>
+                      </nav>
+                    </section>
+                    <aside>
+                
+                <!--responsive/fluid bootstrap images-->
+
+                <div class="row1">
```

Comment:
For column/row layouts, you can use Bootstrap defined classes like row and col.  You can add responsive classes to change the number of columns as the screen gets smaller.

https://getbootstrap.com/docs/5.0/layout/grid/